apollo-router/src/plugins/cache/entity.rs

```diff
@@ -50,6 +50,8 @@ use crate::services::supergraph;
 use crate::spec::TYPENAME;
 use crate::Context;
 
+/// Change this key if you introduce a breaking change in entity caching algorithm to make sure it won't take the previous entries
+pub(crate) const ENTITY_CACHE_VERSION: &str = "0.1";
```

Comment:
It's not a hard feeling, but I feel like we want to version from `1.0`. It sounds out a more confident message.